src/server/services/procedures/phone-iot/phone-iot.js

```diff
@@ -292,6 +299,9 @@ if (PHONE_IOT_MODE === 'native' || PHONE_IOT_MODE === 'both') {
             color: { parse: types.parse.Number, default: this.getColor(66, 135, 245) },
```

Comment:
It would be better to make options a parameterized type rather than adding your own parsing code here.

Reply:
(This would probably be good to add in its own PR.)